spec.md

```diff
@@ -413,6 +419,22 @@ message CreateVolumeResponse {
   }
 }
 
+message Principal {
+  // describes attributes for defining the principal identity of 
+  // a volume.
+
+  // This field is OPTIONAL. If specified it contains the CO namespace
+  // identity in which the the following identity information (user /
+  // group) exists.
+  string namespace = 1;
```

Comment:
@govint Let me state the requirements so that you see where this is coming from:
- For doing access control, filesystems need posix user and group (or SID on Windows), abstracted as "Principal" here.
- The design posits that a container acts as one specific Principal when accessing storage. This information is a part of the container deployment. Hence, the CO must provide this information.
- This implies that there is container -> volume relationship to be established by the SP, not only a machine -> volume as the spec currently assumes.